pkg/core/native/notary.go

```diff
@@ -60,29 +60,29 @@ func newNotary() *Notary {
 		manifest.NewParameter("from", smartcontract.Hash160Type),
 		manifest.NewParameter("amount", smartcontract.IntegerType),
 		manifest.NewParameter("data", smartcontract.AnyType))
-	md := newMethodAndPrice(n.onPayment, 100_0000, callflag.States)
+	md := newMethodAndPrice(n.onPayment, 1<<15, callflag.States)
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("lockDepositUntil", smartcontract.BoolType,
 		manifest.NewParameter("address", smartcontract.Hash160Type),
 		manifest.NewParameter("till", smartcontract.IntegerType))
-	md = newMethodAndPrice(n.lockDepositUntil, 100_0000, callflag.States)
+	md = newMethodAndPrice(n.lockDepositUntil, 1<<15, callflag.States)
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("withdraw", smartcontract.BoolType,
 		manifest.NewParameter("from", smartcontract.Hash160Type),
 		manifest.NewParameter("to", smartcontract.Hash160Type))
-	md = newMethodAndPrice(n.withdraw, 100_0000, callflag.States)
+	md = newMethodAndPrice(n.withdraw, 1<<15, callflag.States)
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("balanceOf", smartcontract.IntegerType,
 		manifest.NewParameter("addr", smartcontract.Hash160Type))
-	md = newMethodAndPrice(n.balanceOf, 100_0000, callflag.ReadStates)
+	md = newMethodAndPrice(n.balanceOf, 1<<15, callflag.ReadStates)
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("expirationOf", smartcontract.IntegerType,
 		manifest.NewParameter("addr", smartcontract.Hash160Type))
-	md = newMethodAndPrice(n.expirationOf, 100_0000, callflag.ReadStates)
+	md = newMethodAndPrice(n.expirationOf, 1<<15, callflag.ReadStates)
```

Comment:
`getMaxNotValidBeforeDelta` and `setMaxNotValidBeforeDelta` should also be adjusted.

Reply:
Fixed.